.github/in-cluster-test-scripts/aks-install.sh

```diff
@@ -6,10 +6,10 @@ set -e
 # Install Cilium
 cilium install \
   --disable-check=az-binary \
-  --azure-subscription-id "${AZURE_SUBSCRIPTION_ID}" \
+  --helm-set=azure.subscriptionID="${AZURE_SUBSCRIPTION_ID}" \
   --azure-node-resource-group "${AZURE_NODE_RESOURCE_GROUP}" \
```

Comment:
I think this should also be moved to
```suggestion
  --helm-set=azure.resourceGroup="${AZURE_NODE_RESOURCE_GROUP}" \
```